<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addequipmentDialog</class>
 <widget class="QDialog" name="addequipmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>179</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qconicalgradient(cx:0, cy:0.011, angle:135.7, stop:0 rgba(255, 64, 0, 69), stop:0.252809 rgba(255, 96, 0, 85), stop:0.45 rgba(255, 117, 0, 200), stop:0.488764 rgba(255, 159, 0, 101), stop:0.606742 rgba(255, 106, 0, 179))
</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0)
</string>
   </property>
   <property name="text">
    <string>Выберите параметры груза:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_warning">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="ConicalGradientPattern">
        <gradient centralx="0.000000000000000" centraly="0.011000000000000" angle="135.699999999999989" type="ConicalGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="69">
           <red>255</red>
           <green>64</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.252809000000000">
          <color alpha="85">
           <red>255</red>
           <green>96</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.450000000000000">
          <color alpha="200">
           <red>255</red>
           <green>117</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.488764000000000">
          <color alpha="101">
           <red>255</red>
           <green>159</green>
           <blue>0</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.606742000000000">
          <color alpha="179">
           <red>255</red>
           <green>106</green>
           <blue>0</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>err</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>110</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Наименование</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="le_name">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 63, 24, 255), stop:0.55 rgba(235, 61, 61, 255), stop:1 rgba(88, 38, 38, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Координата ц.м</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="le_Xm">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 63, 24, 255), stop:0.55 rgba(235, 61, 61, 255), stop:1 rgba(88, 38, 38, 255));</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Масса</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="le_m">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 63, 24, 255), stop:0.55 rgba(235, 61, 61, 255), stop:1 rgba(88, 38, 38, 255));</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addequipmentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
